packages/zipkin-instrumentation-cujojs-rest/test/integrationTest.js

```diff
@@ -1,4 +1,5 @@
-const {Tracer, ExplicitContext} = require('zipkin');
+// const {Tracer, ExplicitContext} = require('zipkin');
+import { Tracer, ExplicitContext } from 'zipkin';
 const express = require('express');
```

Comment:
Could we just change all the requires in the whole project to import, all at once? (I think there are codemods for it)